helm/values.yaml

```diff
@@ -84,6 +84,8 @@ jobDefinitions:
       type: ''
     export:
       type: ''
+    seeding:
```

Comment:
seeding has already known has: "seed" in global values, please change

Reply:
![image](https://github.com/user-attachments/assets/7c3ea540-47a9-4920-a924-1a2cab0d70da)